ximea_ros2_cam/launch/multipleCam.launch.py

```diff
@@ -0,0 +1,54 @@
+import os
+from launch import LaunchDescription
+from launch_ros.actions import Node
+from launch.actions import TimerAction
+from ament_index_python.packages import get_package_share_directory
+
+def generate_launch_description():
+    package_name ="ximea_ros2_cam"
+
+    cam1_config_params = os.path.join(
+        get_package_share_directory(package_name),
+        "config",
+        "xiCam1_config.yaml")
+
+    cam2_config_params = os.path.join(
+        get_package_share_directory(package_name),
+        "config",
+        "xiCam2_config.yaml")
+
+    first_node = Node(
```

Comment:
rename this variable in order to clarify its propose

Reply:
done

Reply:
Commit: https://github.com/Brazilian-Institute-of-Robotics/ximea_ROS2_driver/pull/1/commits/2bb6e0eae9117872aa357f1acf15b0cc064acba2